waspc/data/Generator/templates/server/package.json

```diff
@@ -16,7 +16,8 @@
     "delay": "1000"
   },
   "engines": {
-    "node": ">={= nodeVersion =}"
+    "node": "{= nodeSemverString =}",
```

Comment:
I decided this name was more appropriate because the string now contains a compatibility prefix (in this case, `^`).